api/database/functions.py

```diff
@@ -155,3 +161,11 @@ async def batch_function(function, data, batch_size=100):
     ])
 
     return
+
+
+async def is_valid_rsn(rsn: str) -> bool:
+    return re.fullmatch('[\w\d\s_-]{1,13}', rsn)
```

Comment:
Shouldn't this be {1,12} instead of {1,13?}

Reply:
we have to consider AnonymousUser